src/ExtendableSpeciesCard.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import EbiSpeciesIcon from 'react-ebi-species'
+
+const MAX = 5
+
+class ExtendableSpeciesCard extends React.Component {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isHidden: this.props.content ? this.props.content.length >= MAX : false
+    }
+    this.onClick = this.onClick.bind(this)
+  }
+ 
+  onClick() {
+    this.setState({
+      isHidden: !this.state.isHidden
+    })
+  }
+    
+  render() {
+    const {iconSrc, iconDescription, content} = this.props
+    
+    let visibleContent = (max, content) => {
+      return content.slice(0, max).map((item) => {
+        return item.url ?
+                <li className={`url`} style={{marginBottom: `0.3rem`}} key={item.text}><a href={item.url} key={item.text}>{item.text}</a></li> :
+                <li className={`text`} style={{marginBottom: `0.3rem`}} key={item.text}>{item.text}</li>
+      })
+    }
+
+    return (
+      <div className={`column column-block text-center combo card`} style={{marginBottom:0, paddingBottom: `25px`}}>
+        {
+          iconDescription && <h5 className="species-name">{iconDescription}</h5>
```

Comment:
Be consistent about class name syntax. Our convention is to always use backticks:
```
className={`species-name`}
```

Please review this here and elsewhere in the rest of the code.